### DISCUSSION TITLE
Result stutter

### DISCUSSION THREAD

**Discussion by KeesDraai** [mifi/lossless-cut#1863] [Q&A]
I have a mkv file which I split into several small files. But the small files all show some stuttering.

It looks like after each second there is a stutter like as if 1 frame is missing.

I have used LLC lots of times and havent experienced this before. The original mkv file is okay and shows no stuttering at all.

Anybody have an idea?

**Comment by mifi** [comment]
are you using smart cut? how long is the duration of the files? see also https://github.com/mifi/lossless-cut/blob/master/issues.md

**Reply by Dozer58** [reply]
I'm also getting stutter, not using smart cut just the default, was using an older version so updated to latest to see if that fixed it but still the same.
After looking at both files with media info I see the scan type / interlace method has changed, also cut file has stream size added.

ORIGINAL
Scan type                                : MBAFF
Scan type, store method        : Interleaved fields
Scan order                               : Top Field First
Bits/(Pixel*Frame)                     : 0.256
Language                                 : English


CUT
Scan type                                : Interlaced
Scan type, store method          : Separated fields (2 fields per block)
Scan order                               : Top Field First
Bits/(Pixel*Frame)                    : 0.256
Stream size                              : 1.37 GiB (95%)
Language                                 : English

**Reply by mifi** [reply]
This is probably a bug in ffmpeg in that it’s not preserving interlacing metadata properly. If you like you could try to download latest ffmpeg from ffmpeg.org and then copy the latest ffmpeg command from losslesscut and instead run with the newest version

**Reply by Dozer58** [reply]
I got the latest ffmpeg and ffprobe exe and replaced the ones in losslesscut folder (I didn't have any dll's with the download)
The scan type now says MBAFF in the cut vid same as original but the store method still says Separated fields (2 fields per block) instead of interleaved.

Stutter is still there.

**Reply by mifi** [reply]
ok, then it's not fixed by ffmpeg yet. could report a bug at their bug tracker if you're also able to reproduce it with a minimal command, e.g.
```bash
ffmpeg -i input.mp4 -c copy output.mp4
```

**Comment by KeesDraai** [comment]
Just to show what the problem was, I uploaded a part of the clip to Vimeo. But strangly the problem didn't occur in that Video when I watched it online. So the problem is not LossLessCut, but it must be something in VLC. Because when I look the clip in VLC it 'stutters', but when I look at it online, or with Media Player, it shows no stuttering.
